src/constants/cookies.ts

```diff
@@ -0,0 +1,7 @@
+export const HIDE_FEEDBACK = "HIDE_FEEDBACK";
+export const ANNOUNCEMENT_TOAST = "announcement-toast";
+export const SUBSCRIPTION_METHOD = "subscription-method";
+export const getNotificationTriggerCookie = (type: string) =>
+  `${type}-notification-trigger`;
+export const SLACK_NOTIFICATION_BANNER = "has-closed-slack-banner";
+export const CURRENT_PROJECT = "mci-project-cookie";
```

Comment:
Tiny nit but probably can drop the "-cookie" since we don't use that pattern in the others
```suggestion
export const CURRENT_PROJECT = "mci-project";
```

Reply:
I actually meant to reuse the cookie that we use on the legacy waterfall: https://github.com/evergreen-ci/evergreen/blob/main/service/middleware.go#L26